app/utils/iframeHandler.js

```diff
@@ -1,4 +1,25 @@
-// @flow 
+// @flow
+import { LedgerBridge } from 'yoroi-extension-ledger-bridge';
+
+export async function prepareLedgerBridger(ledgerBridge: LedgerBridge): Promise<void> {
```

Comment:
this is good. I would like to handle a timeout case as welll. What do you think of something like this:

```
_wait = async ms => new Promise((resolve) => setTimeout(resolve, ms));

export async function waitIFrameLoaded(name: string): Promise<void> {
  if (ledgerBridge == null) {
    throw new Error(`Error: LedgerBridge is undefined`);
  }

  const promise = new Promise((resolve, reject) => {
    const start = new Date();
    const maxWait = 10000; // 10 seconds

    while (iframeLoaded) {
      _wait(500);
      iframeLoaded = ledgerBridge.isReady;

      if ((new Date().getTime() - start) > maxWait) {
        reject(new Error(`Time out. IFrame ${name} wasn’t able to load in 10 seconds.`));
      }
    }

    resolve();
  });

  return promise;
}
```